### ISSUE TITLE
Error setting yt-dlp executable in Windows 10

### ISSUE THREAD

**Issue by parasiteoflife** [axcore/tartube#616]
### What operating system are you using?
Windows 10

### What version of Tartube are you using?
2.5.0

### If your bug report is "I can't download this video", please provide a link to the video
After updating to 2.5.0 I started receiving a message that mingw64\bin\yt-dlp.exe was missing when up until now Tartube was using the executable available in my PATH. Anyways I went a set up a custom path for the executable pointing to C:\Windows\yt-dlp.exe and although the "default path" in there points to an executable file too whenever I start a download (I'm using classic mode) I get an error because Tartube wants to load the custom path as if I specified a python file, it tries to run `python3 -X utf8 C:\Windows\yt-dlp.exe` and obviously errors out, please allow us to use the executable available in PATH again.

**Comment by axcore** [comment]
I tested this, but the Classic Mode download was successful. (I moved yt-dlp.exe from ..\mingw64\bin\ to C:\Users\MYNAME\Desktop, and I set the custom path in Tartube preferences).

![tartube-test1](https://github.com/axcore/tartube/assets/37951926/5524ada5-edae-4950-a404-9a6b8ef0266a)
![tartube2](https://github.com/axcore/tartube/assets/37951926/f9fff874-23bd-472f-9f85-4c3b649ead98)

Please show me a screenshot of Tartube's Output tab, when the video fails to download. You could also try downloading [the same video that I tested](https://www.youtube.com/watch?v=jNQXAC9IVRw).

**Comment by axcore** [comment]
`--ffmpeg-location`

This is not supported on MS Windows. Please remove it until we have solved your actual problem. You should also remove all the other unnecessary stuff like --mark-watched.

I suggest that you click **Edit > System preferences > Options > Download options**, and add a new set of download options. When your problem is solved, you can delete these options, and continue using `--mark-watched`, etc.

`C:\Windows`

I am not a Windows programmer, so I am not sure it's a good idea to create new files in this folder. (Maybe it's ok, I don't know - I am not a Windows programmer).

Try doing what I did: put the yt-dlp.exe on your desktop, so the path will be `C:\Users\MYNAME\Desktop\yt-dlp.exe`

**Comment by axcore** [comment]
>You mean in Tartube? Because yt-dlp supports it.
Yes, yt-dlp supports it but MSYS2 does not, for reasons unknown.

**Comment by Bolb389** [comment]
Hello,
I downloaded yt-dlp.exe from [this page (Windows (Win7 SP1+) standalone x64 binary (recommended for Windows))](https://pypi.org/project/yt-dlp/#standalone-py2exe-builds-windows), unfortunately this is still not working. Can you tell me which version can I use on Windows ?

**Comment by KyattPL** [comment]
I just got that issue btw, fresh install of tartube, pointed to a yt-dlp.exe, no custom commands or anything:

![image](https://github.com/user-attachments/assets/ba44dc20-0c31-487e-bb0a-2d229e221e64)

Do I install yt-dlp via python3 globally or what? When I try to switch the downloader file path to a default one I get the following error:

![image](https://github.com/user-attachments/assets/4fc568cf-85fa-4a18-8a5c-0c5aa6758dd5)
